addons/greptimedb/values.yaml

```diff
@@ -12,8 +12,8 @@ clusterDomain: ".cluster.local"
 images:
   pullPolicy: IfNotPresent
   greptimedb:
-    registry: docker.io
-    repository: greptime/greptimedb
```

Comment:
There is also a fixed registry
`docker.io/greptime/greptimedb-initializer:0.1.0-alpha.12`
https://github.com/apecloud/kubeblocks-addons/blob/chore/change_addon_registry/addons/greptimedb/templates/cmpd-datanode.yaml#L89

Multiple busybox missing registries and repositories
https://github.com/apecloud/kubeblocks-addons/blob/chore/change_addon_registry/addons/greptimedb/templates/cmpd-datanode.yaml#L58